test/parallel/test-net-connect-local-error.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+var common = require('../common');
```

Comment:
I believe the plan is to update them as we go. But in general, all new tests should use const wherever possible